compiler+runtime/test/jank/form/quote/pass-dot-prefixed-symbol.jank

```diff
@@ -0,0 +1,3 @@
+(assert (= (symbol ".a") (quote .a)))
+
+:success
```

Comment:
These `.jank` tests are not used for lexing. Your lex tests in C++ cover this.

Reply:
Hmm, cljc support is still in another branch. I'll just drop this one.